src/helpers_format.cpp

```diff
@@ -0,0 +1,43 @@
+#include <bpf/libbpf.h>
```

Comment:
nit: the filename is too generic, it should be `bpf_helpers_format` - also maybe be part of `util` folder?